src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -181,7 +185,7 @@ public void sendAssignToJudgeConfirmationEmailToRespondentSolicitor(FinremCaseDe
      * No Return.
      * <p>Please use @{@link #sendAssignToJudgeConfirmationEmailToIntervenerSolicitor(FinremCaseDetails, SolicitorCaseDataKeysWrapper)}</p>
      *
-     * @param caseDetails instance of CaseDetails
+     * @param caseDetails     instance of CaseDetails
```

Comment:
revert this bit?

Reply:
![Screenshot 2024-10-23 at 13 04 55](https://github.com/user-attachments/assets/213ac6b1-31d1-4ddd-8c63-17fcd9919534)
This was an auto style fix. I think it's fine